lib/davinci_dtr_test_kit/profiles/communication_request/communication_request_read.rb

```diff
@@ -17,7 +18,7 @@ def resource_type
     def communication_request_id_list
       return [nil] unless respond_to? :communication_request_ids
 
-      communication_request_ids.split(',').map(&:strip)
+      communication_request_ids&.split(',')&.map(&:strip)
```

Comment:
I don't think that split will ever return nil, so I don't think you need the second `&` before the call to map.

Reply:
Funny you say this, this is actually something my editor automatically adds on save with prettier, looks like I get the following lint error when I remove it:

<img width="888" alt="image" src="https://github.com/user-attachments/assets/a95886cf-8245-4220-9970-0ea9226aec22">

Happy to still remove it though

Reply:
no, the linter knows better than I do...